client/tests/test_client.py

```diff
@@ -1,11 +1,13 @@
 """Unit tests for client application's CRUD opeartions."""
 
 from django.conf import settings
+from django.contrib.auth.models import User
```

Comment:
As you can see now we have two different files with the same kind of code and tests, one tests using the login credentials and the other one tests using the token based authentication. I want you to create a parent test class in __init__ with all the duplicate code and two child classes in each file which will handle the authentication. 
Ping me for more details.

Reply:
> As you can see now we have two different files with the same kind of code and tests, one tests using the login credentials and the other one tests using the token based authentication. I want you to create a parent test class in **init** with all the duplicate code and two child classes in each file which will handle the authentication.
> Ping me for more details.

@Ayub-Khan done. You can review again.